pennylane/transforms/optimization/optimization_utils.py

```diff
@@ -18,8 +18,19 @@
 import numpy as np
 
 from pennylane.math import abs as math_abs
-from pennylane.math import allclose, arccos, arctan2, cos, get_interface, is_abstract, sin, stack
+from pennylane.math import (
+    allclose,
+    arccos,
+    arctan2,
+    cos,
+    get_interface,
+    is_abstract,
+    sin,
+    stack,
+    sum,
```

Comment:
can you import this as `math_sum` (like `math_abs` above) because `sum` is a builtin and someone might unknowingly use the wrong `sum` in this file?